genisys/main.py

```diff
@@ -66,3 +66,32 @@ def main():
 
 if __name__ == "__main__":
     main()
+
+'''
+import needed modules,
+create new module with options/configs from other branches.
+'''
+def run():
+
+    #netplan.py
+    import modules.netplan as net
+
+    #get settings for functions
+    netplan = net.Netplan()
+    netLocation = net.install_location()
+    netgenerate = net.generate()
+
+    #base.py
+    import modules.base as base
```

Comment:
Base should be treated as an abstract class. It is only meant to be inherited by the other modules, not instantiated directly.